spec/support/solidus_stripe/webhook/request_helper.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Webhook
```

Comment:
Should this be namespaces in `SolidusStripe` as well? or we have reasons not to do that?

Reply:
No reason. Thanks for catching that! Fixed.